composer.json

```diff
@@ -5,14 +5,24 @@
     "keywords": ["php","framework", "upf"],
     "license": "MIT",
     "minimum-stability": "stable",
+    "authors": [
+        {
+            "name": "Eldad Fux",
+            "email": "eldad@appwrite.io"
+        }
+    ],
```

Comment:
Please make sure this isn't in here. There was another PR specifically made to remove authors.

Reply:
Okay I will remove those 4 lines then. Thanks for pointing this out.

Reply:
Please make sure to remove this.